lib/bindings/HTTPBindings.js

```diff
@@ -406,11 +408,70 @@ function start(callback) {
         router: express.Router()
     };
 
+    var options = {
+        swaggerDefinition: {
+            info: {
+                title: 'IoT Agent UL2 - HTTP', // Title (required)
+                version: '2.0.0', // Version (required)
+                description: 'This documentation explains the POST and GET requests to the route /iot/d' // Description (not required)
```

Comment:
The Swagger spec states that the route is always `/iot/d`, however the [config.js](https://github.com/telefonicaid/iotagent-ul/blob/master/config.js#L235) can be overridden by using a non-standard config or applying a Docker env variable. Maybe the **description** should use `config.getConfig().iota.defaultResource` to describe the route. It would be kind of tricky to alter the swagger  **definition** itself.

Reply:
fixed in 43ed8517f0cf88571d9688db7aa47e7c7fb9ed57